src/client.ts

```diff
@@ -378,8 +377,10 @@ export interface ICreateClientOpts {
      * Verification methods we should offer to the other side when performing an interactive verification.
      * If unset, we will offer all known methods. Currently these are: showing a QR code, scanning a QR code, and SAS
      * (aka "emojis").
+     *
+     * See {@link types.VerificationMethod} for a set of useful constants for this parameter.
      */
-    verificationMethods?: Array<VerificationMethod>;
+    verificationMethods?: Array<string>;
```

Comment:
Why is the typing getting looser?

Reply:
you raced with me: see my comment below https://github.com/matrix-org/matrix-js-sdk/pull/4129#discussion_r1535772278.